packages/typespec-client-generator-core/src/http.ts

```diff
@@ -247,6 +248,8 @@ function createContentTypeOrAcceptHeader(
       kind: "constant",
       value: bodyObject.contentTypes[0],
       valueType: type,
+      name: `${httpOperation.operation.name}ContentType`,
```

Comment:
What if we keep it "ContentType"? I'm not sure if we could recognize it if it is not called "ContentType".

Reply:
this is the name for the type of the `Content-Type` parameter, not the `Content-Type` parameter itself. Since each content type parameter can have its own types, if all of the types are called `ContentType` then there are duplicate names for these constant types